docs/src/guides/laravel/deploy/scheduling.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Workers, cron jobs, and task scheduling"
+sidebarTitle: "Scheduling tasks"
+weight: -100
+description: Schedule tasks for your Laravel app.
+---
+
+Laravel offers a very convenient and flexible way of scheduling tasks. A large set of [helper functions](https://laravel.com/docs/scheduling#schedule-frequency-options) help you schedule commands and jobs.
+
+Once the scheduled tasks are defined, they need to be effectively executed at the right time and pace.
+The recommended way is a cron configuration entry running the `artisan schedule:run` command.
+
+```yaml {location=".platform.app.yaml"}
+crons:
+    # Run Laravel's scheduler every 5 minutes, which is as often as crons can run on Professional plans.
+    scheduler:
+        spec: '*/5 * * * *'
+        cmd: 'php artisan schedule:run'
+```
+
+The [minimum time between cron jobs](../../../configuration/app/app-reference.md#cron-job-timing) being triggered depends on your plan. Task scheduling may then be contradicted by the cron minimum frequency. Schedules outside the specified cron frequency will be ignored and the related tasks won't be triggered.
```

Comment:
```suggestion
The [minimum time between cron jobs](../../../configuration/app/app-reference.md#cron-job-timing) being triggered depends on your plan. Task scheduling may then be contradicted by the cron minimum frequency. Schedules outside the specified cron frequency are ignored and the related tasks aren't triggered.
```

We try to avoid the future tense: https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#use-the-present-tense

Reply:
Awesome @CollierCZ. I discover this guide, and I have to make it mine. So great to have it in place!